src/Button/types.ts

```diff
@@ -35,6 +37,10 @@ export type ButtonBaseProps = {
    * Items that are disabled can not be clicked, selected, or navigated through.
    */
   disabled?: boolean
+  /**
+   * Allow button width to fill its container.
+   */
+  block?: boolean
```

Comment:
Me too.. I had it this way originally, but it felt like I was creating unnecessary friction. Since we use block in PVC I think it makes sense to keep it as block here, and maybe in the future do a sweeping change of all `block` props to `fullWidth`